syntax = "proto3";

package test_proto;

message demo_msg {
    int32 id = 1;
    string name = 2;
}


